test/config-utils.spec.js

```diff
@@ -0,0 +1,50 @@
+import { getAccessToken } from '../src/utils/config-utils'
+
+describe('Config Utils', () => {
+  beforeEach(() => {
+    process.env.access_token = null
+  })
+
+  afterEach(() => {
+    process.env.access_token = null
+  })
+
+  describe('getAccessToken', () => {
+    test('should get access token from authInfo', () => {
+      const target = {
+        authInfo: {
+          access_token: 'T0K3N'
+        }
+      }
+
+      const token = getAccessToken(target)
+
+      expect(token).toEqual('T0K3N')
+    })
+
+    test('should throw an error when access token is unavailable', () => {
```

Comment:
In this test case, access token is available, but not valid.

Reply:
I’d still leave it as unavailable, since I think the empty string means the value hasn’t been specified.
To me, invalid means that the access token can’t be parsed, or that it has expired, which we are not checking for here.
But happy to change it if we feel really strongly about it as well. :slightly_smiling_face:

Reply:
empty / undefined / null I would say all mean 'unavailable' (in the sense we're sure a valid token is not available) happy to be overruled though